### PR TITLE
Use types.h min/max over windef.h

### PR DESCRIPTION
**Pull request by ksmit799**
- One step closer to cross platform support.
- Vector War still uses windef.h min/max, not sure what we want to do there.

### CODE REVIEW

File: src/lib/ggpo/types.h [hunk 1]
```diff
@@ -40,16 +40,16 @@ typedef char int8;
 typedef short int16;
 typedef int int32;
 
+
 /*
  * Macros
  */
 #if defined(_DEBUG)
-#define BREAK  DebugBreak();
```
Comment (pond3r) [inline src/lib/ggpo/types.h]: Could you call this `DEBUG_BREAK` instead of just `BREAK`?

File: src/lib/ggpo/types.h [hunk 2]
```diff
@@ -40,16 +40,16 @@ typedef char int8;
...
+#  define BREAK  DebugBreak();
 #else
-#define BREAK  exit(1);
```
Comment (pond3r) [inline src/lib/ggpo/types.h]: If we're not in debug builds, we probably want to just keep going.  The SDK shouldn't ever `exit`, in any case (that should be a decision left up to the application.

File: src/lib/ggpo/types.h (hunk 1 above)
Comment (ksmit799) [reply src/lib/ggpo/types.h]: This was actually already included in types.h (I didn't write it), just fixed the formatting. It isn't used anywhere in the codebase, so it should be safe to just remove it.

File: src/lib/ggpo/types.h (hunk 2 above)
Comment (ksmit799) [reply src/lib/ggpo/types.h]: See above.

**Comment by ksmit799** [comment]
@pond3r I've removed the macro entirely (it's not used anywhere in the codebase).

**Pull request merged**